podcasts/End of Year/Stories/EpilogueStory.swift

```diff
@@ -49,7 +51,28 @@ struct EpilogueStory: StoryView {
 
                 Spacer()
             }
-        }.background(Constants.backgroundColor.allowsHitTesting(false).onAppear(perform: prepareHaptics))
+            .background(
+                ZStack(alignment: .top) {
+                    Color.black
+
+                    background
+                    .offset(x: -geometry.size.width * 0.4, y: -geometry.size.height * 0.22)
+                    .clipped()
+                }
+                .ignoresSafeArea()
+                .allowsHitTesting(false)
+                .onAppear(perform: prepareHaptics)
+            )
+        }
```

Comment:
```suggestion
            .background(
                ZStack(alignment: .top) {
                    Color.black

                    background
                    .offset(x: -geometry.size.width * 0.4, y: -geometry.size.height * 0.22)
                }
                .allowsHitTesting(false)
                .onAppear(perform: prepareHaptics)
            )
        }.clipped().ignoresSafeArea()
```

Reply:
As we talked about on the other PR this affects big screens, so I'll leave this change for another PR if that's ok to you!.